README.md

```diff
@@ -1,4 +1,5 @@
 # Dynamic Wallpaper Engine
+![AUR version](https://img.shields.io/aur/version/plasma5-wallpapers-dynamic)
```

Comment:
whitespace nitpick: can you please add a line between the heading and the image link?